static/build/.tmp_update/script/network/start_smbd.sh

```diff
@@ -1,3 +1,5 @@
+#unset preload or samba doesn't work correctly.
+
 export LD_PRELOAD=""
```

Comment:
*You know*, there's actually a more direct way of doing this:
```sh
unset LD_PRELOAD
```
(I know this was from prev work, just wanted to mention)